lib/dangermattic/plugins/check_tracks.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Danger
+  # Plugin for checking tracks-related changes and providing instructions in the PR if that's the case.
+  class CheckTracks < Plugin
+    DEFAULT_TRACKS_FILES = [
+      'AnalyticsTracker.kt',
+      'AnalyticsEvent.kt',
+      'LoginAnalyticsTracker.kt',
+      'WooAnalyticsStat.swift'
+    ].freeze
+
+    DEFAULT_TRACKS_USE_MATCHERS = [
+      /AnalyticsTracker\.track/
+    ].freeze
+
+    TRACKS_PR_INSTRUCTIONS = <<~MESSAGE
+      This PR contains changes to Tracks-related logic. Please ensure the following are completed:
+      **PR Author**
+      - The PR must be assigned the **Tracks** label
+      **PR Reviewer**
+      - The tracks events must be validated in the Tracks system.
```

Comment:
I feel like one of the "tracks" is redundant in the message. There's enough context to understand that the event is a Tracks event

```suggestion
      - The events must be validated in the Tracks system.
```